src/components/UI/Card.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import './card.css'
+
+const Card = (props) => {
+
+  return <div>
+
+<div className="card">
```

Comment:
This would be a good approach for simple layouts and buttons.
The main purpose of our card component is reusability. Hence, you can see that the user will always define the structure and functionality of a component in the parent component which is something we're trying to avoid.
Consider passing a `product` prop to your card component so that you can have all the details of a product inside the card component. and you won't have to rewrite the content of a card in case you want to reuse your card somewhere else